### DISCUSSION TITLE
[Resolved] Git auto-deploy suddenly stopped working

### DISCUSSION THREAD

**Discussion by mitraraman** [vercel/community#2777] [Help] [answered]
### Summary

I noticed today that my auto-deploy stopped working. The changes are pushing to the main branch no problem and I've made no changes to my vercel setup since yesterday night (when it was last working). I can deploy from CLI, but I want it to auto-deploy whenever there's a new change.

Any ideas?

### Steps to Reproduce

Push a commit to git main branch, no deploy triggered on Vercel.

**Comment by swarnava** [comment]
Hi,

Do you have a Commit SHA which we can look at?

**Comment by mitraraman** [comment]
Sure: https://github.com/mitraraman/chiyoadmin/commit/9908c8af2c9278542baad920ca0c96de61164dcd

Mitra

**Comment by languageleague** [comment]
Same issue here, it seems to be a Vercel problem

**Comment by swarnava** [comment]
Okay, it looks like an issue on GitHub side: https://www.githubstatus.com/incidents/1g1gkh0qpyvs

**Comment by adamvigneault** [comment]
Similar issue, new branches pushed to repo no longer deploy to Vercel (though they do push to Github just fine).

**Comment by swarnava** [comment, marked as answer]
The related GitHub incident has been resolved. https://www.vercel-status.com/incidents/0t59wvr17drc
